spec.bs

```diff
@@ -273,6 +289,9 @@ Each {{URLPattern}} object has an associated <dfn for=URLPattern>hash component<
     1. [=Assert=]: |input| is a {{URLPatternInit}}.
     1. If |baseURL| is given, then throw a {{TypeError}}.
     1. Set |init| to |input|.
+  1. Let |ignoreCase| be false.
```

Comment:
It seems like we don't really need a separate `|ignoreCase|` variable here.  We can just reference `|options|[{{URLpatternOptions/ignoreCase}}]` below.  If we set the default value to false in the webidl, then we don't have to check for existence before referencing it.